### DISCUSSION TITLE
Help with  Github Pages in a fork

### DISCUSSION THREAD

**Discussion by YvoLeVrai** [octokatherine/word-master#39] [Q&A]
Hello,
I am trying to build a version of this website but in french (for friends to be able to play it) but I'm encountering a problem when trying to setup the github pages of this site. Do someone know how to do it  ? I've never used this tool, and I'm either obtaining a blank page or a readme, but can't make the game show up.

Thanks for your help

**Comment by octokatherine** [comment]
Hi! I'm not 100% sure it is exactly the same for a fork, but in general you should

1. enable github pages in the settings tab of your repo ![image](https://user-images.githubusercontent.com/49968061/148654507-dfa995c8-ed18-40b3-b727-aec6ab822ee3.png)
2. in `package.json`, change the `homepage` value `http://{username}.github.io/{repo-name}` where `{username}` is your GitHub username, and `{repo-name}` is the name of the GitHub repository
3. add the GitHub repository as a "remote" in your local git repository `git remote add origin https://github.com/{username}{repo-name}.git`
4. run `npm run deploy`

For reference, I followed [these instructions](https://github.com/gitname/react-gh-pages)

**Reply by YvoLeVrai** [reply]
Thanks for the steps, I feel like it could come from the package.json value that I didn't change. I'll try this and tell you if this work.

**Reply by octokatherine** [reply]
I believe @cmot17 successfully deployed a fork to gh pages if you need more help
